pdc/apps/auth/apps.py

```diff
@@ -4,9 +4,14 @@
 # http://opensource.org/licenses/MIT
 #
 from django.apps import AppConfig
+from django.db.models.signals import post_migrate
 
 
 class AuthConfig(AppConfig):
     name = 'pdc.apps.auth'
     label = 'kerb_auth'
     verbose_name = 'Auth with Kerberos support'
+
+    def ready(self, *args, **kwargs):
+        from . import signals
+        post_migrate.connect(signals.update_resources, sender=self)
```

Comment:
Is it be possible to use `@receiver(post_migrate)` decorator on `update_resources()` instead?

Reply:
It's doable, but requires further code to make sure it runs only once. By default `@receiver` would connect it to all senders, so the code would run for each application that has migrations. Given that we should still import the `signals` module in `ready()` method, I does not seem like a big win.

Reply:
OK, understood.